frontend/pages/sites/$siteId/builds/index.jsx

```diff
@@ -110,28 +79,30 @@ function SiteBuildList() {
               </tr>
             </thead>
             <tbody>
-              {builds.data.map((build) => {
-                const SiteBuild = memo(Build, [build.state]);
+              {Array.isArray(data) &&
```

Comment:
Are there conditions where it can render children with data not being an array? I like the idea of removing conditionals from within the `QueryPage` children and having it handled within the component logic

Reply:
Added `placeholderdata` as an empty array to the query and am using `isPlaceholderData` to decide to render children.